lib/src/support/draw_utils.cpp

```diff
@@ -98,16 +117,19 @@ namespace cycfi { namespace elements
       cnv.fill_style(gradient);
 
       cnv.begin_path();
-      cnv.add_round_rect(bounds.inset(1, 1), corner_radius-1);
+      draw_round_rect(cnv, bounds.inset(1, 1), {corner_radius[0]-1, corner_radius[1]-1, corner_radius[2]-1, corner_radius[3]-1});
```

Comment:
I think `bounds.inset(1, 1)` ought to be a temporary: `auto r = bounds.inset(1, 1)`, which will shorten the line as well.